src/components/alert/alert.tsx

```diff
@@ -237,7 +248,10 @@ export class Alert implements OpenCloseComponent, LocalizedComponent, LoadableCo
             <slot name={SLOTS.message} />
             <slot name={SLOTS.link} />
           </div>
-          {queueCount}
+          <div class={CSS.actionsEnd} hidden={!hasEndActions}>
```

Comment:
I think you will need to add the following CSS...

```css
[hidden] {
  @apply hidden;
}
```

I needed this for the `calcite-panel` to work correctly.

https://github.com/Esri/calcite-components/pull/5129#discussion_r942875228